discovery_child_development/__init__.py

```diff
@@ -34,5 +34,8 @@ def get_yaml_config(file_path: Path) -> Optional[dict]:
 logger = logging.getLogger(__name__)
 
 # base/global config
-_base_config_path = Path(__file__).parent.resolve() / "config/config.yaml"
```

Comment:
Separated out base and config, in case we ever wanted to use the base config file!